subsys/usb/Kconfig

```diff
@@ -79,6 +79,12 @@ config USB_DEVICE_REMOTE_WAKEUP
 	help
 	  This option requires USBD peripheral driver to also support remote wakeup.
 
+config USB_DEVICE_DRIVER_HANDLE_CTRL_EPIN_ZLP
```

Comment:
I would rather go for a name like:
USB_DEVICE_DISABLE_STACK_ZLP or similar to explicitly show that this option disable ZLP handling in zephyr usb stack.

Reply:
No, because it does not describe why.

Reply:
`USB_DEVICE_ZLP_EPIN_HW` or `USB_DEVICE_DISABLE_ZLP_EPIN_HANDLING`

Reply:
changed to USB_DEVICE_DISABLE_ZLP_EPIN_HANDLING